app/models/programme_type.rb

```diff
@@ -15,11 +15,13 @@
 class ProgrammeType < ApplicationRecord
   extend Mobility
   include TransifexSerialisable
+  include TranslatableAttachment
```

Comment:
thank you for putting together this concern @urbanwide 🙏  (nice implementation too!)  made implementing the multi file uploads super simple :)

Reply:
Yay, glad it has been useful :-)

Reply:
it really was!  now I just need to do the same for activity_categories, activity types, intervention type groups, and intervention types :D

Reply:
Hehe good luck :-)